.travis.yml

```diff
@@ -5,10 +5,11 @@ python:
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+env:
+  - TOXENV=flake8
 install:
-  - "pip install flake8"
-  - "pip install -Ur requirements.txt"
-before_script: "flake8 ."
-script: "py.test -v --driver Firefox --variables variables.json tests"
```

Comment:
I gave this a stab - I /think/ it's actually being passed in, but as there are new (hopefully unrelated and are rather from this repo last being touched 8 months ago).